qiita_pet/test/test_admin_processing_job_handlers.py

```diff
@@ -45,5 +45,30 @@ def test_get_missing_argument(self):
                       response.body.decode('ascii'))
 
 
+class TestSampleValidation(BaseAdminTests):
+    def test_get(self):
+        response = self.get('/admin/sample_validation/')
+        self.assertEqual(response.code, 200)
+
+    def test_post(self):
+        post_args = {
+            'qid': 1,
+            'snames': 'SKB1.640202 SKB2.640194 BLANK.1A BLANK.1B'
+        }
+        response = self.post('/admin/sample_validation/', post_args)
+        self.assertEqual(response.code, 200)
+        snames = ['SKB1.640202', 'SKB2.640194', 'BLANK.1A', 'BLANK.1B']
+        body = response.body.decode('ascii')
+        for name in snames:
+            self.assertIn(name, body)
+
```

Comment:
@sarayupai Good job! But note that the test is called 'test_post' and you're testing positive and negative outcomes in this test. It's not obvious to the reader why this test should return 500. You have to compare each input to realize the only difference is 2 and perhaps that's an invalid id. It would be good to add a comment here summarizing what you're doing and what you expect to see happen.

Reply:
Added comments indicating the positive and negative tests